.github/workflows/test.yml

```diff
@@ -21,12 +23,10 @@ jobs:
           if [ "$TASK" = "flake8" -o "$TASK" = "flake8-wip" ]; then pip install flake8; fi
           # If this causes SSL errors, then Sourceforge is probably in disaster recovery mode and needing Javascript. Could switch to a specific mirror, e.g. kent.dl.sourceforge.net
           if [ "$TASK" = "pychecker" -o "$TASK" = "pychecker-wip" ]; then pip install http://sourceforge.net/projects/pychecker/files/pychecker/0.8.19/pychecker-0.8.19.tar.gz/download; fi
-          #if [ "$TASK" = "codespell" ]; then pip install git+https://github.com/codespell-project/codespell.git; fi
-          if [ "$TASK" = "codespell" ]; then pip install codespell; fi
-          sudo apt-get install xvfb
-          sudo apt-get install libhtml-parser-perl
-          sudo apt-get install lintian
-          sudo apt-get install moreutils
+          if [ "$TASK" = "codespell" ]; then sudo apt-get -y install python3-pip git; fi
+          if [ "$TASK" = "codespell" ]; then python3 -m pip install --no-input git+https://github.com/codespell-project/codespell.git; fi
+          #if [ "$TASK" = "codespell" ]; then pip install codespell; fi
+          sudo apt-get install -y xvfb libhtml-parser-perl lintian moreutils
```

Comment:
Having this in one line makes the diff less readable in the future, should packages removed or added. However, one `apt install` command per line is also not perfect.
Can we do:
```
apt install \
  xvfb \
  ... \
  ... \
  ...
```
?